### PR TITLE
Upgrade Backtest class

### PR DESCRIPTION
**Pull request by alexander-gr**
Fix bugs in the Backtest class.
Remove unused code and improve syntax.
Add visualization method.

### CODE REVIEW

File: san/backtest.py [hunk 1]
```diff
@@ -2,53 +2,44 @@
 import pandas as pd
 import numpy as np
 import san
+import matplotlib.pyplot as plt
```
Comment (tspenov) [inline san/backtest.py]: Should we put `matplotlib` in https://github.com/santiment/sanpy/blob/master/setup.py#L20 ?

**Comment by tspenov** [comment]
@alexander-gr Maybe we should add some usage info in the README ?

**Comment by tspenov** [comment]
@alexander-gr You can also bump the version, so we can release it

File: san/backtest.py (hunk 1 above)
Comment (alexander-gr) [reply san/backtest.py]: Yep, sure
Thanks!

**Comment by alexander-gr** [comment]
> @alexander-gr Maybe we should add some usage info in the README ?

As far as I know Jan is working on all docs ATM

**Pull request merged**